samples/templates/default-azuredeploy.json

```diff
@@ -161,7 +161,7 @@
         "sqlSchemaAutomaticUpdatesEnabled": {
             "type": "string",
             "allowedValues":["auto", "tool"],
-            "defaultValue": "[if(equals(parameters('solutionType'),'FhirServerCosmosDB'), 'tool', '')]",
+            "defaultValue": "[if(equals(parameters('solutionType'),'FhirServerCosmosDB'), 'tool', 'auto')]",
```

Comment:
Same as above, we want to leave this blank.

Reply:
Reverted this as well, thanks!